Integrations/src/test/java/io/deephaven/integrations/learn/ComputerTest.java

```diff
@@ -0,0 +1,126 @@
+package io.deephaven.integrations.learn;
+
+import io.deephaven.db.v2.InMemoryTable;
+import io.deephaven.db.v2.sources.ColumnSource;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+public class ComputerTest {
+
+    private static InMemoryTable table;
+    private static Input[] inputs;
```

Comment:
I think this goes away.

Reply:
Table? Inputs? Both? I see `inputs` going away, but it seems like I need the table.

Reply:
inputs